### PR TITLE
[#53] Develop "add/edit entity" template + pages(first commit for che…

### PR DESCRIPTION
**Pull request by Potyomkin-Igor**
[#53] Develop "add/edit entity" template + pages(first commit for check in)

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/addPages/add_investment.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+<div class="container">
+    <div class="row">
+        <div class="col-xs-12 col-sm-10 col-md-8 col-lg-6 col-lg-offset-3" style="margin-top: 50px;">
+            <div class="panel-card panel-success-card">
+                <div class="text-center"><h3>Add an investment</h3></div>
+                <div class="panel-body">
+                    <div class="row">
+                        <div class="col-xs-12">
+                            <div class="text-center">
+                                <p align="center">Here you can add you investment in <u>Project</u></p>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/addPages/add_investment.html]: MAJOR Remove this deprecated "align" attribute. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AS1827)

File: src/main/webapp/WEB-INF/views/addPages/add_investment.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                                <p align="center">Total investment needed: <u>100000</u>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/addPages/add_investment.html]: MAJOR Remove this deprecated "align" attribute. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AS1827)

File: src/main/webapp/WEB-INF/views/addPages/add_investment.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                                    <span class="fa fa-usd" aria-hidden="true"></span>
+                                </p>
+                                <p align="center">Minimal investment: <u>5000</u>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/addPages/add_investment.html]: MAJOR Remove this deprecated "align" attribute. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AS1827)

File: src/main/webapp/WEB-INF/views/addPages/add_invinterest.html [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+                                <label class="col-md-4 control-label" for="category">Category</label>
+                                <div class="col-xs-12 col-md-6 col-lg-8" style="margin-bottom: 25px">
+                                    <input id="category" name="category" type="text" placeholder="Category"
+                                           class="form-control">
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="col-md-4 control-label " for="cost">Cost</label>
+                                <div class="col-xs-12 col-md-6 col-lg-8" style="margin-bottom: 20px">
+                                    <!--<input id="cost" name="cost" type="number" style="width: 300px "  placeholder="Cost"-->
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/addPages/add_invinterest.html]: MAJOR Remove this commented out code. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)

File: src/main/webapp/WEB-INF/views/editPages/edit_invinterest.html [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+                                <label class="col-md-4 control-label" for="category">Category</label>
+                                <div class="col-xs-12 col-md-6 col-lg-8" style="margin-bottom: 25px">
+                                    <input id="category" name="category" type="text" placeholder="Category"
+                                           class="form-control">
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="col-md-4 control-label " for="cost">Cost</label>
+                                <div class="col-xs-12 col-md-6 col-lg-8" style="margin-bottom: 20px">
+                                    <!--<input id="cost" name="cost" type="number" style="width: 300px "  placeholder="Cost"-->
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/editPages/edit_invinterest.html]: MAJOR Remove this commented out code. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 5 issues
* ![MAJOR][MAJOR] 5 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by GetFire** [comment]
Я бы все таки переделал их под модальные окна. Но так тоже нормально. Не забудь обновить ветку

**Comment by semyonov-m-y** [comment]
Всё вроде бы ок. Вот только почему ДВА раза отображается add_invinterest.html в файлах? - 160 и 139 строк в них

File: src/main/webapp/WEB-INF/views/addPages/add_event.html [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+    </div>
+</div>
+<div class="container" style="margin-top: 10%;">
+    <div class="row">
+        <div class="col-xs-12 col-sm-10 col-md-8 col-lg-6 col-lg-offset-3">
+            <div class="panel-card panel-success-card">
+                <div class="text-center"><h3>Create event</h3></div>
+                <div class="panel-body">
+                    <div class="row">
+                        <div class="col-xs-12">
```
Comment (zozich) [inline src/main/webapp/WEB-INF/views/addPages/add_event.html:60]: What's the point? 12 is default value

File: src/main/webapp/WEB-INF/views/addPages/add_event.html [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+                            <div class="form-group">
+                                <label class="col-md-4 control-label" for="name">Name</label>
+                                <div class="col-xs-12 col-md-6 col-lg-8" style="margin-bottom: 25px">
```
Comment (zozich) [inline src/main/webapp/WEB-INF/views/addPages/add_event.html]: Is it intentional that on larger screens it is larger than on medium ones?

File: src/main/webapp/WEB-INF/views/addPages/add_event.html [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="modal fade" id="modal-login">
```
Comment (zozich) [inline src/main/webapp/WEB-INF/views/addPages/add_event.html]: Why d you have modal login inline? You should have one template for all pages

File: src/main/webapp/WEB-INF/views/addPages/add_event.html [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+                <button class="btn btn-danger" type="button" data-dismiss="modal">Close</button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.js"></script>
+<script src="js/bootstrap.js"></script>
+<script src="js/myJs.js"></script>
+<script src="js/salvattore.min.js"></script>
```
Comment (zozich) [inline src/main/webapp/WEB-INF/views/addPages/add_event.html]: Do you really need it?

File: src/main/webapp/WEB-INF/views/addPages/add_investment.html [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="modal fade" id="modal-login">
```
Comment (zozich) [inline src/main/webapp/WEB-INF/views/addPages/add_investment.html]: What is login form doing here?

**Review by zozich — CHANGES_REQUESTED** [review]
Amend everywhere

File: src/main/webapp/WEB-INF/views/addPages/add_event.html (hunk 6 above)
Comment (Potyomkin-Igor) [reply src/main/webapp/WEB-INF/views/addPages/add_event.html:60]: but, if I explicitly specify this parameter, the form as it becomes more collected

**Review by zozich — APPROVED** [review]
OK

**Pull request merged**